### PR TITLE
Format `nodes-base` package (A-F)

### PR DESCRIPTION
**Pull request by michael-radency**

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -6,81 +7,81 @@ packages/design-system
...
-packages/nodes-base/nodes/Flow
-packages/nodes-base/nodes/FormIo
-packages/nodes-base/nodes/Formstack
-packages/nodes-base/nodes/Freshdesk
-packages/nodes-base/nodes/Freshservice
-packages/nodes-base/nodes/FreshworksCrm
-packages/nodes-base/nodes/Ftp
-packages/nodes-base/nodes/Function
-packages/nodes-base/nodes/FunctionItem
+# packages/nodes-base/nodes/ActionNetwork
```
Comment (ivov) [inline .prettierignore:10]: On completion, let's remove instead of commenting out.

File: .prettierignore [hunk 2]
```diff
@@ -271,3 +272,4 @@ packages/nodes-base/nodes/Zendesk
 packages/nodes-base/nodes/Zoho
 packages/nodes-base/nodes/Zoom
 packages/nodes-base/nodes/Zulip
+packages/nodes-base/nodes/Mindee/GenericFunctions.ts
```
Comment (ivov) [inline .prettierignore:275]: Why is this needed?

File: packages/core/src/NodeExecuteFunctions.ts [hunk 3]
```diff
@@ -537,9 +537,8 @@ function digestAuthAxiosConfig(
 async function proxyRequestToAxios(
 	uriOrObject: string | IDataObject,
 	options?: IDataObject,
+	// tslint:disable-next-line:no-any
```
Comment (ivov) [inline packages/core/src/NodeExecuteFunctions.ts:540]: This file is in `core` and does not belong here.

Comment (michael-radency) [reply packages/core/src/NodeExecuteFunctions.ts]: restored from master

File: packages/nodes-base/nodes/Mindee/Mindee.node.ts [hunk 4]
```diff
@@ -167,7 +167,9 @@ export class Mindee implements INodeType {
 						const rawData = this.getNodeParameter('rawData', i) as boolean;
 
 						if (items[i].binary === undefined) {
-							throw new NodeOperationError(this.getNode(), 'No binary data exists on item!', { itemIndex: i });
+							throw new NodeOperationError(this.getNode(), 'No binary data exists on item!', {
```
[the remark this answers is not in the document]
Comment (michael-radency) [reply packages/nodes-base/nodes/Mindee/Mindee.node.ts]: restored from master

File: .prettierignore (hunk 1 above)
Comment (michael-radency) [reply .prettierignore]: done

File: .prettierignore (hunk 2 above)
Comment (ivov) [reply .prettierignore]: But you need to add `packages/nodes-base/nodes/Mindee` to the ignored dirs.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.189.0`
